doc/source/data/creating-datasets.rst

```diff
@@ -304,6 +304,26 @@ are supported for each of these storage systems.
     :start-after: __read_parquet_az_begin__
     :end-before: __read_parquet_az_end__
 
+.. tabbed:: SQL Databases
+
+  Read data from databases that provide a Python DB API2-compliant connector.
+
+  .. testcode::
```

Comment:
Let's also add example for MySQL, PostgreSQL, Snowflake, Databricks and BigQuery. It's ok that the example is not runnable, but give user the sense how to use the API and library they need to import.